WordPress/Classes/ViewRelated/Stats/SiteStatsDashboardViewController.swift

```diff
@@ -178,7 +187,14 @@ private extension SiteStatsDashboardViewController {
                                                        animated: false)
             }
 
-            periodTableViewController.selectedDate = periodDate ?? StatsDataHelper.currentDateForSite()
+            if let newDate = periodDate {
```

Comment:
Review these changes... I think I will probably roll this back.

This was added to try and not overwrite the selected date that is read from user defaults.